src/javascript/crypto/e2e/extension/ui/settings/settings.js

```diff
@@ -338,17 +339,18 @@ ui.Settings.prototype.importKeyring_ = function(file) {
  * Renders the UI elements needed for requesting the passphrase of an individual
  * PGP key.
  * @param {string} uid The UID of the PGP key.
- * @param {!function(string)} callback The callback to invoke when the
- *     passphrase has been provided.
+ * @return {!e2e.async.Result<string>} A promise resolved with the user-provided
+ *     passphrase.
  * @private
  */
-ui.Settings.prototype.renderPassphraseCallback_ = function(uid, callback) {
+ui.Settings.prototype.renderPassphraseCallback_ = function(uid) {
```

Comment:
This method is duplicated in src/javascript/crypto/e2e/extension/ui/setup/setup.js and should be updated there as well to use the new callback API.